src/plugins/GUITestBase/src/GTUtilsDashboard.h

```diff
@@ -59,7 +81,14 @@ class GTUtilsDashboard {
     static void clickOutputFile(HI::GUITestOpStatus &os, const QString &outputFileName);
 
     static QString getTabObjectName(Tabs tab);
+
     static bool hasNotifications(HI::GUITestOpStatus &os);
+
+    static QList<Notification> getNotifications(HI::GUITestOpStatus &os);
+
+    // Returns a QString containing all dashboard notifications, splitted by '\n'
+    static QString getJoinedNotificationsString(HI::GUITestOpStatus &os);
```

Comment:
This method is not needed at all: getNotificationList(os).join("\n") can be used instead

Reply:
What is `getNotificationList()`?

Reply:
I meant getNotifications() method. But you are right, there is no join(",") method there.